src/components/CartButton/CartButton.css

```diff
@@ -0,0 +1,28 @@
+.red-badge {
+	height: 15px;
+	width: auto;
+	min-width: 11px;
+	padding-left: 2px;
+	padding-right: 2px;
+	white-space: nowrap;
+	border-radius: 8px;
+	background: #ff0000;
+	position: absolute;
+	top: 0px;
+	right: 0px;
+	margin-right: -3px;
+	margin-top: -3px;
+	font-size: 11px;
+	line-height: 15px;
+	text-align: center;
+	vertical-align: middle;
+}
+
+.cart-wrapper {
+	top: 20px;
+  	right: 30px;
+	position: absolute;
+	display: flex;
+	background-image: '../../img/shopping_cart.svg';
+	flex-direction: row;
```

Comment:
since you're absolutely positioning the indicator button, you probably don't need `display: flex` or `flex-direction` here